setup.py

```diff
@@ -123,7 +123,7 @@ def cythonize(x, compile_time_env=None):
                 library_dirs=['/usr/lib', '/usr/local/lib'] + LD_LIBRARY_PATH,
                 libraries=libraries,
                 language="c++",
-                extra_compile_args=["-std=c++11"] + CFLAGS,
+                extra_compile_args=["-std=c++14"] + CFLAGS,

```

Comment:
Just double checking @trevor-vincent, but it looks like this change causes _all_ platforms to use `C++14`? Is it possible to branch here so that only Windows uses C++14, or should it not make too much difference?

Reply:
As far as I know MSVC just ignores the std=c++XX flags lol, so should we just set to std=c++11?

Reply:
Yeah let's give it a go!
```suggestion
                extra_compile_args=["-std=c++11"] + CFLAGS,
```